tox.ini

```diff
@@ -49,6 +50,14 @@ commands =
   mypy: mypy --namespace-packages opentelemetry-api/src/opentelemetry/
 ; For test code, we don't want to enforce the full mypy strictness
   mypy: mypy --namespace-packages --config-file=mypy-relaxed.ini opentelemetry-api/tests/
+; For the SDK, we don't require type annotations
+  mypy-extra: -mypy --namespace-packages --config-file=mypy-minimal.ini opentelemetry-sdk/src/opentelemetry
+  mypy-extra: -mypy --namespace-packages --config-file=mypy-minimal.ini opentelemetry-sdk/tests/
+; Same for extensions
+  mypy-extra: -mypy --namespace-packages --config-file=mypy-minimal.ini ext/opentelemetry-ext-wsgi/src/opentelemetry/ext/
+;  mypy-extra: -mypy --namespace-packages --config-file=mypy-minimal.ini ext/opentelemetry-ext-wsgi/tests/
```

Comment:
Why leave these in but commented?

Reply:
WIP -- I might comment them back in later. If not, I'll of course add a comment explaining why they are disabled.